src/ServiceInsight/Saga/SagaResources.xaml

```diff
@@ -25,4 +25,13 @@
     
     <SolidColorBrush x:Key="FailedMessageBackground" Color="#F9916F" />
     <SolidColorBrush x:Key="RetriedMessageBackground" Color="#F2F2F2" />
+
+    <Style x:Key="GrayNullTextStyle" TargetType="{x:Type TextBlock}">
```

Comment:
Yes, good point. It is more like 'ItalicWhenNull' style due to the Trigger. I'll rename it.